pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -3517,7 +3526,10 @@ private Topic getTopicReference(TopicName topicName) {
         } catch (RestException e) {
             throw e;
         } catch (Exception e) {
-            throw new RestException(e);
+            if (e.getCause() instanceof NotAllowedException) {
```

Comment:
`NotAllowedException` is related to security problems (authorisation).

why are we rewriting this to "CONFLICT" ? shouldn't it be "FORBIDDEN" ?